state-chain/node/src/chain_spec/sisyphos.rs

```diff
@@ -158,4 +158,4 @@ pub fn extra_accounts() -> Vec<(AccountId, AccountRole, FlipBalance, Option<Vec<
 pub const BITCOIN_SAFETY_MARGIN: u64 = 5;
 pub const ETHEREUM_SAFETY_MARGIN: u64 = 6;
 pub const ARBITRUM_SAFETY_MARGIN: u64 = 1;
-pub const SOLANA_SAFETY_MARGIN: u64 = 1; //TODO: put correct value
+pub const SOLANA_SAFETY_MARGIN: u64 = 1; // Unused - we use "finalized" instead
```

Comment:
It's currently used in the chain spec as we have a safety margin for all chains. I'd also remove it but that should be part of another PR.

Reply:
We don't have one for polkadot (set to None in the chainspec). It's not important but if we're going to change it we might as well change it something correct, otherwise it's just confusing.

Reply:
Sure, I just updated the comment because the old one was just wrong. It's true that we should clean it up but at least this comment is not wrong.